crates/edr_generic/src/receipt/execution.rs

```diff
@@ -32,25 +25,16 @@ impl
         header: &edr_eth::block::PartialHeader,
         transaction: &crate::transaction::SignedWithFallbackToPostEip155,
         result: &ExecutionResult<l1::HaltReason>,
-        hardfork: l1::SpecId,
+        _hardfork: l1::SpecId,
```

Comment:
is the hardfork needed by other chains?

Reply:
Potentially, if those changes still support querying the `root` (or some other hardfork-specific receipt type)